import-export-cli/impl/apiProducts.go

```diff
@@ -80,5 +83,27 @@ func GetAPIProductList(accessToken, unifiedSearchEndpoint, query, limit string)
 	} else {
 		return 0, nil, errors.New(string(resp.Body()))
 	}
+}
 
+// Creates the initial api_product_param.yaml in the given file path
+//	The file will be populated with environments and default import parameters for "vcs deploy".
+func ScaffoldAPIProductParams(file string) error {
+	envs := utils.GetMainConfigFromFile(utils.MainConfigFilePath)
+	tmpl, _ := box.Get("/init/api_product_params.tmpl")
```

Comment:
Logic below this line is duplicated in all scaffolding functions. We can extract to make it a reusable function to prevent code duplication.

Reply:
Replaced all scaffolding functions with a common function ScaffoldParams https://github.com/wso2/product-apim-tooling/pull/412/commits/93cd1a77638acf9ee748c7b4042442ac30f40f50